packages/editor/src/utils/index.js

```diff
@@ -2,5 +2,6 @@
  * Internal dependencies
```

Comment:
We have some prior art in having a component in @wordpress/editor that adds/enhances props to a component in @worpress/components, e.g., https://github.com/WordPress/gutenberg/blob/master/packages/editor/src/components/panel-color/index.js and we implemented them in src/components maybe we can also add this one there to be consistent.

Reply:
Done!